src/Core/Components/Menu/FluentMenu.razor

```diff
@@ -3,7 +3,7 @@
 <CascadingValue Value="this" IsFixed="true">
     @if (Open)
     {
-        <FluentOverlay OnClose="@CloseAsync" Transparent="true" />
+        <FluentOverlay @bind-Visible=Open OnClose="@CloseAsync" Transparent="true" FullScreen="true" />
```

Comment:
It's more by "naming convention", but could you always use double-quotes with attributes? `@bind-visible="@Open"`